code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -21,11 +21,17 @@
 	open_sound_volume = 35
 	close_sound_volume = 50
 	drag_slowdown = 0
+	door_anim_squish = 138 //in this context the azimuth angle for over 90 degree
```

Comment:
You don't "re-use" variables. Make a new variable, like `azimuth_angle_2`

Reply:
ok i defined a new var instead